packages/wrangler/src/dev/create-worker-preview.ts

```diff
@@ -206,15 +214,19 @@ export async function createPreviewSession(
 			token: string;
 			prewarm: string;
 		};
-		const inspector = new URL(inspector_websocket);
-		inspector.searchParams.append("cf_workers_preview_token", token);
-
+		let inspector: URL | undefined;
```

Comment:
NIT: this is a URL to request the inspector not the inspector itself...


```suggestion
		let inspectorUrl: URL | undefined;
```